build.sh

```diff
@@ -194,28 +194,9 @@ generate_event_logging_sources()
     fi
 
 # Event Logging Infrastructure
-   __GeneratedIntermediate="$__IntermediatesDir/Generated"
-   __GeneratedIntermediateEventProvider="$__GeneratedIntermediate/eventprovider_new"
-   __GeneratedIntermediateEventPipe="$__GeneratedIntermediate/eventpipe_new"
-
-    if [[ -d "$__GeneratedIntermediateEventProvider" ]]; then
-        rm -rf  "$__GeneratedIntermediateEventProvider"
```

Comment:
Do we no longer want to remove the old stuff?

Reply:
That was an artifact of using a temp directory, writing the files there, the moving them to the intermediate directory. That is not needed since it now holds the file in memory until it is ready to write